airbrake/notifier.py

```diff
@@ -92,24 +108,27 @@ def __repr__(self):
 
     @property
     def context(self):
-        """The python, os, and app environment context."""
+        """System, application, and user data to make more sense of errors."""
         if not self._context:
-            self._context = {}
-            # python
             version = platform.python_version()
-            self._context.update({'language': 'Python %s' % version})
-            # os
             plat = platform.platform()
-            self._context.update({'os': plat})
-            # env name
-            self._context.update({'environment': self.environment})
-            self._context.update({'hostname': self.hostname})
-
-            # TODO(samstav)
-            #   add user info:
-            #       userID, userName, userEmail
-            #   add application info:
-            #       version, url, rootDirectory
+
+            self._context = {
+                'notifier': self.notifier,
+                'os': plat,
+                'hostname': self.hostname,
+                'language': 'Python %s' % version,
```

Comment:
Should be `"Python/%s"` to conform our scheme that we use for official notifiers.